docs/NewAPI.md

```diff
@@ -101,11 +101,9 @@ val consumer = consumerProvider.apply() //6
 
 Here it is an example of producer provider usage:
 ```scala
-val producerProvider = ProducerProvider[Array[Byte], String](
-    "localhost:9092",
-    new ByteArraySerializer(),
-    new StringSerializer()
-).props("some.props", "value")
+val producerProvider = 
+    ProducerProvider[Array[Byte], String]("localhost:9092", new ByteArraySerializer, new StringSerializer)
+    .props("some.props" -> "value")
```

Comment:
Changed the layout of the sample to match that of the consumer - to show they are similar.